src/css/application.css

```diff
@@ -604,3 +604,21 @@ body {
 .u__hidden {
   display: none;
 }
+
+.loading {
+  align-items: center;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  width: 100%;
+}
+
+.loading__spinner {
+  background-image: url('/images/spinner.gif');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  cursor: not-allowed;
+  height: 100px;
+  width: 100px;
+}
```

Comment:
Looks like your text editor doesn’t automatically add a newline at the end of the file—mind adding one here? Or it might be worth changing your editor settings to always do it since I think that’s generally the preference : )